doc/api/fs.md

```diff
@@ -4133,7 +4133,7 @@ This happens when:
 <!-- YAML
 added: v0.0.2
 changes:
-  - version: REPLACEME
+  - version: v17.9.0
```

Comment:
Shouldn't this be semver-major?

Reply:
It was added on v17.x-staging with https://github.com/nodejs/node/pull/42613/commits/99f9880a6fd0f26afd7411e677efb79c3a625c05#diff-11c13307cb334211e0d11f4e7e43bdc0a34e437779a63a6ff645e62448d973d2R4136-R4140, probably an issue with the cherry-picking, it doesn't belong there.

Reply:
The diff to fix the issue is

```diff
diff --git a/doc/api/fs.md b/doc/api/fs.md
index 337baafc13..e6d0927492 100644
--- a/doc/api/fs.md
+++ b/doc/api/fs.md
@@ -4133,15 +4133,6 @@ This happens when:
 <!-- YAML
 added: v0.0.2
 changes:
-  - version: REPLACEME
-    pr-url: https://github.com/nodejs/node/pull/41678
-    description: Passing an invalid callback to the `callback` argument
-                 now throws `ERR_INVALID_ARG_TYPE` instead of
-                 `ERR_INVALID_CALLBACK`.
-  - version: v14.12.0
-    pr-url: https://github.com/nodejs/node/pull/34993
-    description: The `buffer` parameter will stringify an object with an
-                 explicit `toString` function.
   - version: v14.0.0
     pr-url: https://github.com/nodejs/node/pull/31030
     description: The `buffer` parameter won't coerce unsupported input to
```

Reply:
> It was added on v17.x-staging with [99f9880#diff-11c13307cb334211e0d11f4e7e43bdc0a34e437779a63a6ff645e62448d973d2R4136-R4140](https://github.com/nodejs/node/commit/99f9880a6fd0f26afd7411e677efb79c3a625c05#diff-11c13307cb334211e0d11f4e7e43bdc0a34e437779a63a6ff645e62448d973d2R4136-R4140), probably an issue with the cherry-picking, it doesn't belong there.

Yes! It is, I had my doubts, that's not a "semver-major" change, but some hunks are applicable and some of them not. I will apply the diff! Thank you

Reply:
Fixed! Thanks for reviewing!